src/Components/DonationFormModal.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import '../Styles/DonationFormModal.scss';
+
+const DonationFormModal = () => {
+  const [formState, setFormState] = useState({
+    firstName: '',
```

Comment:
Question... I know I was the one to suggested the route to doing a hook like this, but other sources have them as their own const.. Such as const [firstName, setFirstName] = useState('').  I am not sure which looks cleaner